conanfile.py

```diff
@@ -93,21 +94,15 @@ def package(self):
             autotools = self._configure_autotools()
             with tools.chdir(os.path.join(self._source_subfolder, "src", "common")):
                 autotools.install()
+            with tools.chdir(os.path.join(self._source_subfolder, "src", "include")):
+                autotools.install()
             with tools.chdir(os.path.join(self._source_subfolder, "src", "interfaces", "libpq")):
                 autotools.install()
-            self.copy(pattern="*.h", dst="include", src=os.path.join(self._build_subfolder, "include"))
+            with tools.chdir(os.path.join(self._source_subfolder, "src", "bin", "pg_config")):
+                autotools.install()
+            shutil.rmtree(os.path.join(self.package_folder, "include", "postgresql", "server"))
```

Comment:
you could use `tools.rmdir` just a remind for a next PR ;)

Reply:
Ow, didn't kmow about this one, sorry! And thanks for the info